circle.yml

```diff
@@ -16,4 +16,6 @@ deployment:
     branch: master
     commands:
       - turbolift -u $RAXUSER -a $RAXAPIKEY --os-rax-auth $RAXREGION upload -s $HOME/$CIRCLE_PROJECT_REPONAME/app -c $CONTAINER
-      - cd $HOME/$CIRCLE_PROJECT_REPONAME/app/js && touch app2.js && sed 's%https://api.tnyu.org/v2%https://api.tnyu.org/v2-test%g' app.js > app2.js && rm app.js && mv app2.js app.js && turbolift -u $RAXUSER -a $RAXAPIKEY --os-rax-auth $RAXREGION upload -s $HOME/$CIRCLE_PROJECT_REPONAME/app -c "intranet-test"
+      - cd $HOME/$CIRCLE_PROJECT_REPONAME/app/js && touch app2.js && sed 's%https://api.tnyu.org/v2%https://api.tnyu.org/v2-test%g' app.js > app2.js && rm app.js && mv app2.js app.js
+      - cd $HOME/$CIRCLE_PROJECT_REPONAME/app/js && touch app2.js && sed 's%var ApiKey = "";%var ApiKey = prompt("Provide an API key please!");%g' app.js > app2.js && rm app.js && mv app2.js app.js
```

Comment:
@ethanresnick - does not break things if it's empty! I checked :) 

The find and replace will definitely work unless the code-base is changed! They use it in the linux project! But yes, I agree it's a freaking silly solution.